usage-updater/index.js

```diff
@@ -0,0 +1,21 @@
+const Rx = require('rx');
+const Notifications = require('./lib/Notifications');
+const Lambda        = require('./lib/Lambda');
+const UsageRequest  = require('./lib/UsageRequest');
+
+
+exports.handler = function(event, context) {
```

Comment:
Do we want any monitoring/CloudWatch for this? I'm not particularly convinced we do, but worth making it a conscious decision.